README.md

```diff
@@ -83,26 +83,26 @@ any form of iterator, different iterators have to be handled differently.
 
 | Method                      | Rust | Python | npm Itertools | C# |
 | --------------------------- | ---- | ------ | --------------| -- |
-| all                         | ☑    | ☐      | ☑             | ☑  |
-| any                         | ☑    | ☐      | ☑             | ☑  |
+| all                         | ☑    | ☑      | ☑             | ☑  |
+| any                         | ☑    | ☑      | ☑             | ☑  |
 | chain                       | ☑    | ☑      | ☑             | ☑  |
 | collect                     | ☑    | ☐      | ☐             | ☐  |
 | count                       | ☑    | ☑      | ☑             | ☑  |
 | cycle                       | ☑    | ☑      | ☑             | ☐  |
-| enumerate                   | ☑    | ☐      | ☑             | ☐  |
-| filter                      | ☑    | ☐      | ☑             | ☑  |
+| enumerate                   | ☑    | ☑      | ☑             | ☐  |
+| filter                      | ☑    | ☑      | ☑             | ☑  |
 | filterMap                   | ☑    | ☐      | ☐             | ☐  |
 | find                        | ☑    | ☐      | ☑             | ☑  |
 | findMap                     | ☑    | ☐      | ☐             | ☐  |
 | flatMap                     | ☑    | ☐      | ☑             | ☑  |
 | flatten                     | ☑    | ☐      | ☐             | ☐  |
 | forEach                     | ☑    | ☐      | ☐             | ☐  |
 | last                        | ☑    | ☐      | ☐             | ☑  |
-| map                         | ☑    | ☐      | ☑             | ☑  |
-| max                         | ☑    | ☐      | ☑             | ☑  |
-| min                         | ☑    | ☐      | ☑             | ☑  |
+| map                         | ☑    | ☑      | ☑             | ☑  |
+| max                         | ☑    | ☑      | ☑             | ☑  |
+| min                         | ☑    | ☑      | ☑             | ☑  |
 | nth                         | ☑    | ☐      | ☐             | ☑  |
-| partition                   | ☑    | ☑      | ☐             | ☑  |
+| partition                   | ☑    | ☐      | ☐             | ☑  |
```

Comment:
`partition` is neither a built-in function nor a part of the built-in `itertools` module. It belongs to the external module `more-itertools`.